frontend/pulse3d/src/pages/upload-form.js

```diff
@@ -457,13 +469,18 @@ export default function UploadForm() {
       <Uploads>
         <Header>Run Analysis</Header>
         {tabSelection === "Analyze New Files" ? (
-          <FileDragDrop // TODO figure out how to notify user if they attempt to upload existing recording
-            handleFileChange={(files) => setFiles(Object.values(files))}
-            dropZoneText={dropZoneText}
-            fileSelection={files.length > 0 ? files.map(({ name }) => name).join(", ") : "No files selected"}
-            setResetDragDrop={setResetDragDrop}
-            resetDragDrop={resetDragDrop}
-          />
+          <>
+            <FileDragDrop // TODO figure out how to notify user if they attempt to upload existing recording
+              handleFileChange={(files) => setFiles(Object.values(files))}
+              dropZoneText={dropZoneText}
+              fileSelection={
+                files.length > 0 ? files.map(({ name }) => name).join(", ") : "No files selected"
+              }
+              setResetDragDrop={setResetDragDrop}
+              resetDragDrop={resetDragDrop}
+            />
+            <UploadCreditUsageInfo> Each file succesfully uploaded will use 1 credit.</UploadCreditUsageInfo>
```

Comment:
successfully*

Reply:
I think this should say something more along the lines of "Each successfully uploaded file will have analysis run on it, consuming 1 analysis credit each"